core/vm/instructions.go

```diff
@@ -563,12 +567,17 @@ func opGas(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte
 }
 
 func opCreate(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
+	if interpreter.readOnly {
+		return nil, ErrWriteProtection
+	}
+
```

Comment:
Please remove empty line. I've learnt (the hard way) that the golang purists on the team disapprove of empty lines _after_ a closing bracket, with the reasoning that the bracket is enough of a divider. 

_Unless_ the empty line precedes a comment, in which case it might be ok, because it gives the comment more spotlight.

Reply:
This case, `opCreate` I think doesn't really follows the rules you mention, as it does have empty lines in seemingly irregular places. I'll remove the empty line as you suggested though.